lib/types/timestamp.rs

```diff
@@ -13,6 +13,14 @@ use casper_types::bytesrepr::{self, ToBytes};
 #[derive(Copy, Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash, Debug)]
 pub struct Timestamp(u64);
 
+impl Timestamp {
+    /// Returns the timestamp of the current moment.
+    pub fn now() -> Self {
+        let millis = SystemTime::UNIX_EPOCH.elapsed().unwrap().as_millis() as u64;
```

Comment:
Would it be feasible to handle the error instead of panicking with `unwrap()`?
Getting an `Err` from `elapsed()` is unlikely, but it may still happen if someone messes with the system clock.

Reply:
Done in [c5cdb6a](https://github.com/casper-ecosystem/casper-client-rs/pull/16/commits/c5cdb6afe51143fcf058ac6184e67d21233cd776).